### PR TITLE
nift 2.4.12

### PR DESCRIPTION
**Pull request by chenrui333**
Created with `brew bump-formula-pr`.

### CODE REVIEW

**Comment by carlocab** [comment]
Previous attempt: https://github.com/Homebrew/homebrew-core/pull/74866

See, in particular: https://github.com/Homebrew/homebrew-core/pull/74866#issuecomment-816598625 (i.e. this should use Hombrew LuaJIT instead of a bundled one.)

**Review by carlocab — CHANGES_REQUESTED** [review]
Let's fix this to use a Homebrew LuaJIT.

File: Formula/nift.rb [hunk 1]
```diff
@@ -11,8 +11,11 @@ class Nift < Formula
     sha256 cellar: :any_skip_relocation, mojave:   "62b96189d61ca0c360aa7c00718f0995fde9d20ad81c5a700836083800093aef"
   end
 
+  depends_on "luajit-openresty"
+
   def install
-    system "make"
+    inreplace "Lua.h", "/usr/local/include", Formula["luajit-openresty"].opt_include
+    system "make", "BUNDLED=0", "LUAJIT_VERSION=2.1"
```
Comment (nandahkrishna) [inline Formula/nift.rb:18]: Unfortunately, they've hardcoded `/usr/local/include` in `Lua.h` so I used `inreplace`, for now. Wasn't sure if this should be patched upstream instead.

The other thing is that I'm not able to get `2.1` from the `luajit-openresty` Formula, maybe there's a method I'm not aware of.

**Comment by nandahkrishna** [comment]
ARM (during testing):
```
==> /opt/homebrew/Cellar/nift/2.4.12/bin/nsm init .html
Error: nift: failed
An exception occurred within a child process:
  BuildError: Failed executing: /opt/homebrew/Cellar/nift/2.4.12/bin/nsm init .html
```

**Review by carlocab — APPROVED** [review]
No ARM bottle to begin with, so I'm not sure the ARM failure should be blocking.

**Pull request closed without merging**